### PR TITLE
Updating ReadMe and SEQ Column bugfix

### PR DESCRIPTION
**Pull request by hellothisisMatt**
Updating the read me with information on the Quality Checking module.
Fixed the SEQ column for FASTAQ analysis as it disappeared from match_results.tab

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -91,7 +91,7 @@ If you run ``hansel -h``, you should see the following usage statement:
 
           Subtype Salmonella enterica genomes using 33bp k-mer typing schemes.
           Includes schemes for Heidelberg and Enteritidis subtyping.
-          Developed by Genevi�ve Labb�, James Robertson, Peter Kruczkiewicz, Chad R. Laing, Kim Ziebell, Marisa Rankin, Aleisha R. Reimer, Lorelee Tschetter, Gary Van Domselaar, Eduardo N. Taboada, Sadjia Bekal, Kimberley A. MacDonald, Linda Hoang, Linda Chui, Danielle Daignault, Durda Slavic, Frank Pollari, E. Jane Parmley, Philip Mabon, Elissa Giang, Lok Kan Lee, Jonathan Moffat, Joanne MacKinnon, Benjamin M. Hetman, Roger Johnson, John H.E. Nash.
+          Developed by Genevieve Labbe, James Robertson, Peter Kruczkiewicz, Chad R. Laing, Kim Ziebell, Marisa Rankin, Aleisha R. Reimer, Lorelee Tschetter, Gary Van Domselaar, Eduardo N. Taboada, Sadjia Bekal, Kimberley A. MacDonald, Linda Hoang, Linda Chui, Danielle Daignault, Durda Slavic, Frank Pollari, E. Jane Parmley, Philip Mabon, Elissa Giang, Lok Kan Lee, Jonathan Moffat, Joanne MacKinnon, Benjamin M. Hetman, Roger Johnson, John H.E. Nash.
```
Comment (peterk87) [inline README.rst]: Unicode not displaying properly for you? Looks fine in master.

File: README.rst [hunk 2]
```diff
@@ -156,59 +207,76 @@ Analysis of a single FASTA file
...
+    negative3200083-2.1 NODE_3_length_427905_cov_48.1477_ID_5 3200083 2.1 False SRR1002850_smalltestdata  /home/mgopez/hansel_test_data/old_test_data/SRR1002850_smalltestdata.fasta  heidelberg  0.5.0 PASS
+    negative3204925-2.2.3.1.5 NODE_3_length_427905_cov_48.1477_ID_5 3204925 2.2.3.1.5 False SRR1002850_smalltestdata  /home/mgopez/hansel_test_data/old_test_data/SRR1002850_smalltestdata.fasta  heidelberg  0.5.0 PASS
+    negative3230678-2.2.2.1.1.1 NODE_3_length_427905_cov_48.1477_ID_5 3230678 2.2.2.1.1.1 False SRR1002850_smalltestdata  /home/mgopez/hansel_test_data/old_test_data/SRR1002850_smalltestdata.fasta  heidelberg  0.5.0 PASS
+    negative3233869-2.1.1.1.1 NODE_3_length_427905_cov_48.1477_ID_5 3233869 2.1.1.1.1 False SRR1002850_smalltestdata  /home/mgopez/hansel_test_data/old_test_data/SRR1002850_smalltestdata.fasta  heidelberg  0.5.0 PASS
+    negative3254229-2.2.3.1.3 NODE_3_length_427905_cov_48.1477_ID_5 3254229 2.2.3.1.3 False SRR1002850_smalltestdata/home/mgopez/hansel_test_data/old_test_data/SRR1002850_smalltestdata.fasta  heidelberg  0.5.0 PASS
+    [Following lines ommitted.]
+
+
+Contents of ``tech_results.tab``:
```
Comment (peterk87) [inline README.rst]: Show tech results before other results?

File: README.rst (hunk 1 above)
Comment (hellothisisMatt) [reply README.rst]: Shows question marks where the acccent aigu and grave are supposed to be.

File: README.rst (hunk 2 above)
Comment (hellothisisMatt) [reply README.rst]: I'll change the order

File: README.rst (hunk 1 above)
Comment (peterk87) [reply README.rst]: Yeah looks like it in the diff, but maybe it's alright when rendered to HTML from RST?

Comment (hellothisisMatt) [reply README.rst]: Perhaps, even using `hansel -h` results in the accents missing :(

**Comment by hellothisisMatt** [comment]
This is an old pull request, going to close it as it's outdated.

**Pull request closed without merging**